qtop.py

```diff
@@ -2037,6 +2050,20 @@ class InvalidScheduler(Exception):
     QTOPPATH = os.path.dirname(realpath(sys.argv[0]))  # dir where qtop resides
     SAMPLE_FILENAME = 'qtop_sample_${USER}%(datetime)s.tar'
     SAMPLE_FILENAME = os.path.expandvars(SAMPLE_FILENAME)
+    if options.REPLAY:
+        watch_start_datetime_obj = get_date_obj_from_str(options.REPLAY[0])
+        REC_FP_ALL = '/tmp/qtop_results_%s/*_partview*.out' % os.path.expandvars('${USER}')
+        rec_files = glob.iglob(REC_FP_ALL)  # [::-1] sort after eliminating irrelevants, dummy!!
+        useful_frames = []
+        for f in rec_files:
+            # captured_fp_datetime = f.rsplit('_partview')[1][:-4]  # one way to do it?
+            f_last_modified_date = datetime.datetime.fromtimestamp(os.path.getmtime(f))  # better?
+            if abs(watch_start_datetime_obj - f_last_modified_date) < datetime.timedelta(minutes=int(options.REPLAY[1])):
+                useful_frames.append(f)
+            # if datetime.datetime.now() - f_last_modified_date < datetime.timedelta(minutes=60):  # how old should it be?
+        useful_frames = iter(useful_frames[::-1])
+        REC_FP = '/tmp/qtop_results_%s/*_partview_%s.out' % (os.path.expandvars('${USER}'), options.REPLAY[0])
```

Comment:
`/tmp/` is now hardwired. Could we make that more tunable generic, because some people would rather prefer `/dev/shm`!